_dev/src/ts/pages/UpdatePageUpdateOptions.ts

```diff
@@ -64,6 +65,14 @@ export default class UpdatePageUpdateOptions extends StepPage {
   readonly #onSubmit = async (event: Event) => {
     event.preventDefault();
 
-    await api.post(this.#form.dataset.routeToSubmit!, new FormData(this.#form));
+    const data = new FormData(this.#form);
+
+    analytics.track('[SUE] Update options configured', {
+      disable_all_overrides: Boolean(data.get('PS_DISABLE_OVERRIDES')),
+      disable_non_native_modules: Boolean(data.get('PS_AUTOUP_CUSTOM_MOD_DESACT')),
+      keep_customized_email_templates: Boolean(data.get('PS_AUTOUP_REGEN_EMAIL'))
```

Comment:
If you want to keep using primitive types:

```suggestion
      disable_all_overrides: !!data.get('PS_DISABLE_OVERRIDES'),
      disable_non_native_modules: !!data.get('PS_AUTOUP_CUSTOM_MOD_DESACT'),
      keep_customized_email_templates: !!data.get('PS_AUTOUP_REGEN_EMAIL')
```